server/rpc/interceptors/interceptors.go

```diff
@@ -279,6 +279,8 @@ func subdomainUnaryServerInterceptor() grpc.UnaryServerInterceptor {
 }
 
 func addInvocationIdToLog(ctx context.Context) context.Context {
+	ctx = bazel_request.ExtractValuesIntoContext(ctx)
```

Comment:
I wonder if it would make sense to do it in interceptors, since we are also calling GetInvocationID in propagateInvocationIDToSpanUnaryServerInterceptor

Reply:
Good call; done.